### PR TITLE
Remove community page.

### PR DESCRIPTION
**Pull request by DanVanAtta**
- Gets rid of one more tab header
- We have most of the links on the community page on our website, just missing the A&A.org link. A dedicated navigation tab and page to add the one link in is a stretch to justify, it's very important we keep the website relatively clean.

### CODE REVIEW

File: community.html [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-title: Community
-permalink: /community/
----
-<p>To chat with players and browse live games, download the game and join the lobby.</p>
-<p>Without the lobby, you can join the <a href="https://gitter.im/triplea-game/social">gitter discussion channel</a></p>
-<div class="community-col-wrapper">
-  <div class="community-col">
-    <a href="https://github.com/triplea-game/triplea">
-      <h2>Open Source on Github</h2>
-      <img src="../images/mark-github.svg" alt="Github Icon">
```
Comment (RoiEXLab) [inline community.html]: I don't think those images are being used anywhere else in the webpage, so try removing them as well please. 
The travis build fails if an image that's referenced somewhere gets removed, so we're on the safe side.

Comment (DanVanAtta) [reply community.html]: I could not find any usages either, removed in: 3d907cd

File: community.html [hunk 2]
```diff
@@ -1,34 +0,0 @@
...
-    <a href="https://forums.triplea-game.org/">
-      <h2>TripleA Forum</h2>
-      <img src="../images/icon_menu.png" alt="TripleA icon">
-    </a>
-    <p>Community website with ladder, tournaments, and forums for finding matches and discussing game strategy.</p>
-  </div>
-  <div class="community-col">
-    <a href="https://www.axisandallies.org/">
-      <h2>AxisAndAllies.org</h2>
-      <img src="../images/aaorg-logo.png" alt="Axis &amp; Allies Icon">
```
Comment (RoiEXLab) [inline community.html]: Sorry for not clarifying enough: I meant all image references on this page.

File: community.html [hunk 3]
```diff
@@ -1,34 +0,0 @@
...
-    <p>
-      TripleA is an open source project hosted on <a href="https://github.com/triplea-game/triplea">Github</a>.
-      Material on getting started as a map maker can be found in the
-      <a href="https://github.com/triplea-game/triplea/tree/master/docs/map_making">map making documentation</a>.
-    </p>
-  </div>
-  <div class="community-col">
-    <a href="https://forums.triplea-game.org/">
-      <h2>TripleA Forum</h2>
-      <img src="../images/icon_menu.png" alt="TripleA icon">
```
Comment (RoiEXLab) [inline community.html]: Here as well

**Comment by DanVanAtta** [comment]
@RoiEXLab merging now, we can follow up with any unused images in a new branch easily enough

**Pull request merged**